src/__tests__/Search/SearchView.test.tsx

```diff
@@ -58,8 +57,11 @@ describe('Search View', () => {
 
   it('renders a skip link that sends the focus directly to search container', async () => {
     renderComponent();
-    await waitFor(() => userEvent.tab());
-    await waitFor(() => userEvent.keyboard('{Enter}'));
+
+    // set delay to null to prevent test time-out due to useFakeTimers
+    const user = userEvent.setup({ delay: null });
+    await user.tab();
+    await user.keyboard('{Enter}');
```

Comment:
I'm not sure why, but this was making a _later_ test in this file fail.

This was incorrect too: `waitFor` should be used with things that throw like expectations. In this case I believe it was working around the fact that we use fake timers, because `waitFor` internally advance the fake timeouts. I'm not sure either why it was failing before this change, my guess is some conflicts with how testing-library's waitFor interacts with fake timers...